app/gateway/2.6.x/configure/auth/kong-manager/workspaces.md

```diff
@@ -73,21 +73,26 @@ navigate to the new Workspace's dashboard.
 
     ![New Dashboard](https://doc-assets.konghq.com/1.3/manager/workspaces/02-workspace-dashboard.png)
 
-
 ## Delete or Edit a Workspace
 
-To delete a Workspace, everything inside the
-Workspace must be deleted first. This includes default Roles on the "Admins"
-page.
+To be able to delete a workspace, *all data* must first be deleted from the workspace:
```

Comment:
I think this section needs a header. Maybe "Prepare a workspace for deletion"?

Reply:
I like something like that.